blocks/video-promo-block/features/video-promo/default.jsx

```diff
@@ -18,36 +20,58 @@ const DescriptionText = styled.p`
 `;
 
 const AlertBadge = styled.span`
+  background-color: #db0a07;
+  border-radius: 5px;
+  color: #fff;
   display: inline-block;
   padding: 0.3rem 0.8rem;
-  color: #FFFFFF;
-  background-color: #DB0A07;
-  border-radius: 5px;
 `;
 
 const VideoPromo = ({ customFields }) => {
   const { arcSite } = useFusionContext();
   const {
-    uuid,
     autoplay = false,
-    ratio = 0.5625,
-    title,
-    description,
+    inheritGlobalContent = true,
```

Comment:
do we default true or false of inheritGlobalContent? based on the proptypes, it looks like we already have `true` 

```suggestion
    inheritGlobalContent,
```
![Screen Shot 2020-11-19 at 14 23 08](https://user-images.githubusercontent.com/5950956/99720086-cf732400-2a72-11eb-940f-7af30f1fa89e.png)